sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/Custom/SecurityAssessmentMetadataData.cs

```diff
@@ -1,15 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// <auto-generated/>
-
 #nullable disable
 
 using System.Collections.Generic;
 using Azure.Core;
 using Azure.ResourceManager.Models;
 using Azure.ResourceManager.SecurityCenter.Models;
 
+[assembly: CodeGenSuppressType("SecurityAssessmentMetadataData")]
```

Comment:
Why do we need to suppress the generation of all the model? Can we only customize the deserialzation method?